web/src/app/pages/profile/profilePage.controller.js

```diff
@@ -58,7 +56,7 @@
 
             $http({
                 method: 'POST',
-                url: 'http://localhost:3000/dev/users/getUser',
+                url: IG().api+'/dev/users/getUser',
```

Comment:
Its good if you can create a decorator function instead of appending to URL
Current code
 IG().api+'/dev/users/getUser'

Recommended code
IG.api('/dev/users/getUser')